internal/executor/container_examples/egress_server/cmd/server/main.go

```diff
@@ -0,0 +1,70 @@
+package main
```

Comment:
I'd move this into the root of the directory like `cmd/executor-server`, and move the Dockerfile into the root directory. It can both serve as an example if someone wants to write their own Custom Executor, or they contribute to our own executor.